### PR TITLE
HPCC-12918 Visualize WU Stats

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-12918

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

**Comment by ghalliday** [comment]
The page itself looks good.  Doesn't seem to work with firefox, but works with IE and chrome.

**Comment by GordonSmith** [comment]
Added Work around for FF Issue.

**Comment by ghalliday** [comment]
Verified that works.

File: esp/src/eclwatch/WUStatsWidget.js [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+############################################################################## */
+define([
+    "dojo/_base/declare",
+    "dojo/_base/lang",
+    "dojo/i18n",
+    "dojo/i18n!./nls/hpcc",
+    "dojo/_base/array",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: array not used.

File: esp/src/eclwatch/WUStatsWidget.js [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+    "dojo/_base/Deferred",
+    "dojo/dom",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: dom's not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+############################################################################## */
+define([
+    "dojo/_base/declare",
+    "dojo/_base/lang",
+    "dojo/i18n",
+    "dojo/i18n!./nls/hpcc",
+    "dojo/_base/array",
+    "dojo/_base/Deferred",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: deferred not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+    "dojo/dom",
+    "dojo/dom-construct",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: dom-construct not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 5]
```diff
@@ -0,0 +1,371 @@
...
+    "dojo/on",
+    "dojo/html",
+
+    "dijit/registry",
+    "dijit/Dialog",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: dialog not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 6]
```diff
@@ -0,0 +1,371 @@
...
+
+    "dojox/html/entities",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: entities not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 7]
```diff
@@ -0,0 +1,371 @@
...
+    "dojo/_base/declare",
+    "dojo/_base/lang",
+    "dojo/i18n",
+    "dojo/i18n!./nls/hpcc",
+    "dojo/_base/array",
+    "dojo/_base/Deferred",
+    "dojo/dom",
+    "dojo/dom-construct",
+    "dojo/on",
+    "dojo/html",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: html not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 8]
```diff
@@ -0,0 +1,371 @@
...
+
+    "dojox/html/entities",
+
+    "hpcc/_Widget",
+    "hpcc/ESPBase",
+    "hpcc/GraphWidget",
+    "hpcc/JSGraphWidget",
+    "hpcc/ESPUtil",
+    "hpcc/ESPWorkunit",
+    "hpcc/TimingTreeMapWidget",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: TimingTreeMapWidget not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 9]
```diff
@@ -0,0 +1,371 @@
...
+    "dojo/on",
+    "dojo/html",
+
+    "dijit/registry",
+    "dijit/Dialog",
+
+    "dojox/html/entities",
+
+    "hpcc/_Widget",
+    "hpcc/ESPBase",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: ESPBase not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 10]
```diff
@@ -0,0 +1,371 @@
...
+    "hpcc/GraphWidget",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: GraphWidget not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 11]
```diff
@@ -0,0 +1,371 @@
...
+    "hpcc/JSGraphWidget",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: JSGraphWidget not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 12]
```diff
@@ -0,0 +1,371 @@
...
+    "hpcc/ESPUtil",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: ESPUtil not used

File: esp/src/eclwatch/WUStatsWidget.js [hunk 13]
```diff
@@ -0,0 +1,371 @@
...
+    "hpcc/ESPWorkunit",
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUStatsWidget.js]: ESPWorkunit not used

**Comment by miguelvazq** [comment]
Review complete. Just the unneeded libraries.

**Comment by GordonSmith** [comment]
Removed unused references as per comments, thx.

**Pull request merged**